frontend/src/metabase/entities/users.js

```diff
@@ -126,8 +125,6 @@ const Users = createEntity({
     }
     return state;
   },
-
-  forms,
```

Comment:
This felt nice

Reply:
like this?

![c381a645cd812bb03b6a8708ba472e25_w200](https://github.com/metabase/metabase/assets/30528226/9d88d88f-25a5-4e14-b40a-54f71ac62073)